sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -186,11 +197,17 @@ private boolean shouldApplyScopeData(
 
     final List<SentryEnvelopeItem> envelopeItems = new ArrayList<>();
 
+    Boolean isFatal = false;
```

Comment:
You can use `SentryEvent#isCrashed()` instead, this is how you know if an event is fatal/crashed or not.

Reply:
- [ ] replace isFatal with isCrashed

Reply:
decided to go with DiskFlushNotification check instead